app/assets/javascripts/static/ContactUsView.js

```diff
@@ -0,0 +1,244 @@
+define([
+  'jquery',
+  'backbone'
+], function($, Backbone) {
+
+  'use strict';
+
+  const topics = {
+    'report-a-bug-or-error-on-gfw': {
```

Comment:
I tried to change the topic selector several times but the placeholder doesn't change, is that the expected behaviour?